lib/timeline/Core.js

```diff
@@ -248,16 +248,16 @@ Core.prototype._create = function (container) {
   function handleDrop(event) {
     // return when dropping non-vis items 
     try {
-      var itemData = JSON.parse(event.dataTransfer.getData("text/plain"))
```

Comment:
Yeah I see it now. I'm not sure if I should do this in 2 iterations. It might be worth accepting this PR and I'll submit another PR with IE10 support